vcpkg.json

```diff
@@ -46,20 +47,23 @@
 			"default-features": false
 		},
 		{
-			"name": "lilv",
-			"default-features": false
```

Comment:
I don't understand this change. Why removing lilv and lv2?

Reply:
Oops, my mistake